torch/_inductor/codecache.py

```diff
@@ -825,6 +829,15 @@ def __init__(
                         (kernel_source, constant_args, configs)
                     )
 
+                if "saved_tensors_hooks" in name:
+                    _idx = 0 if "_pack" in name else 1
+                    _list_to_append = self.saved_tensors_hooks_fx_wrap_cache_hashes[
+                        _idx
+                    ]
+                    for node in module.graph.nodes:
+                        if node.meta and node.meta.get("is_wrapped", False):
+                            _list_to_append.append(node.meta["user_cache_hash"])
```

Comment:
maybe cc @jamesjwu  - I remember the logic to generate a cache key being mostly shared across the AOT and inductor caches. In this case, these `saved_tensor_hooks` only show up in the AOT graph, and disappear before inductor. I'm not sure if it's easier though just to add the extra logic to compute cache keys off these graphs in the inductor logic, or somewhere else in the AOT caching logic (the latter seems nicer?)

Reply:
Instead of putting it in FxGraphCacheDetails, would it make more sense to put it in AOTAutogradCacheDetails, so that it's aot specific? AOTAutogradCacheDetails extends FxGraphCacheDetails